src/elements/content-explorer/ContentExplorer.js

```diff
@@ -213,6 +214,7 @@ class ContentExplorer extends Component<Props, State> {
         initialPageSize: DEFAULT_PAGE_SIZE,
         contentPreviewProps: {
             contentSidebarProps: {},
+        contentUploaderProps: {},
```

Comment:
This should not be nested in, should be at the same level as contentPreviewProps

Reply:
Corrected